corehq/warehouse/management/commands/create_batch.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management import BaseCommand, CommandError
+from dimagi.utils.parsing import string_to_utc_datetime
+from corehq.warehouse.models import BatchRecord
+
+
+USAGE = """Usage: ./manage.py create_batch <batch_id> -s <start_datetime> -e <end_datetime>
+"""
+
+
+class Command(BaseCommand):
+    """
+    Example: ./manage.py create_batch 222617b9-8cf0-40a2-8462-7f872e1f1344 -s 2017-05-01 -e 2017-06-01
```

Comment:
i think the start timestamp would always just be the end timestamp of the previous batch run, right? (since we always pull start date exclusive, end date inclusive) otherwise it might be hard for an external process to keep track of the last successful run in the event the system is down or something

similarly, should we be setting end date automatically?